### DISCUSSION TITLE
Plugin that hides "Dismantle" option on all Zulrah-related equipment

### DISCUSSION THREAD

**Discussion by eeznir** [runelite/runelite#16664] [Ideas]
I accidentally Dismantled a Serp Helm into 20k Scales on the GIM the other day while uncharging a bunch of items to be able to put them into group storage. This was quite frustrating all around, and my team members were not happy.

There are already existing plugins that do exactly this, but for different items, such as "Not Empty" removing the "Empty" option on all drinkable potions https://runelite.net/plugin-hub/show/not-empty

or "Lootbag Utilities" which removes the "Destroy" option on all storage containers (Rune Pouch, Herb Sack, etc.) https://runelite.net/plugin-hub/show/lootbag-utilities

**I would like to see a plugin made that hides the "Dismantle" option on the following items**:

- Serpentine helm (uncharged)
- Toxic blowpipe (empty)
- Uncharged toxic trident
- Uncharged toxic trident (e)

**Comment by swirle13** [comment]
https://runelite.net/plugin-hub/show/hotkeyable-menu-swaps has a "Custom Hides" section where you can use regex to remove any options from a right click menu.

You can type in `Dismantle,Toxic Blowpipe*,` or `Dismantle,Serpentine helm*,`. The asterisk is a wildcard, which will make it work for both charged and uncharged, just in case, or you can do `Dismantle,*,` to remove the dismantle option for any items, which is a bit broad and blunt and may not be desired.
